src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/commands.py

```diff
@@ -0,0 +1,58 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from ._util import (get_mysql_servers_operation, get_mysql_databases_operations,
+                    get_mysql_firewall_rules_operations,
+                    get_postgresql_servers_operation, get_postgresql_databases_operations,
+                    get_postgresql_firewall_rules_operations,
+                    ServiceGroup, create_service_adapter)
+
+def load_commands(server_type, servers_operation, firewall_rules_operations, databases_operations):
+    ## server
+    server_sa = create_service_adapter(
+        'azure.mgmt.rdbms.{}.operations.servers_operations'.format(server_type),
+        'ServersOperations')
+
+    with ServiceGroup(__name__, servers_operation, server_sa) as s:
+        with s.group('{} server'.format(server_type)) as c:
+            c.command('create', 'create_or_update')
+            c.command('update', 'update')
+            c.command('delete', 'delete')
+            c.command('show', 'get')
+            c.command('list', 'list_by_resource_group')
+
+    ## firewall rule
+    firewall_rule_sa = create_service_adapter(
+        'azure.mgmt.rdbms.{}.operations.firewall_rules_operations'.format(server_type),
+        'FirewallRulesOperations')
+
+    with ServiceGroup(__name__, firewall_rules_operations, firewall_rule_sa) as s:
+        with s.group('{} server firewall-rule'.format(server_type)) as c:
+            c.command('create', 'create_or_update')
+            c.command('update', 'create_or_update')
+            c.command('delete', 'delete')
+            c.command('show', 'get')
+            c.command('list', 'list')
+
+    ## database
+    database_sa = create_service_adapter(
+        'azure.mgmt.rdbms.{}.operations.databases_operations'.format(server_type),
+        'DatabasesOperations')
+
+    with ServiceGroup(__name__, databases_operations, database_sa) as s:
+        with s.group('{} database'.format(server_type)) as c:
```

Comment:
In sql we have this command group called `db` and I strongly prefer `db` since it is substantially shorter for an important command group.